releases/pulpcore/3.6/settings

```diff
@@ -0,0 +1,3 @@
+FOREMAN_VERSION='none'
```

Comment:
This can live in `releases/pulpcore/settings` since that's loaded prior to foreman.

Reply:
Nope:
https://github.com/theforeman/theforeman-rel-eng/blob/d8af5d364b2ab5c5040489eecca8e5d181a252e1/settings#L13-L17

We'd have to change that loading logic, which I tried to avoid for now.

Reply:
Ah, line 13 is RELEASEDIR and not PROJECTDIR. It should be safe to load that as well, right? We load it around line 50 as well.

Reply:
Yeah, I made a `load_settings` function now, how is that?